src/CLU/Microsoft.ScenarioTests.CLU/out.txt

```diff
@@ -0,0 +1,33 @@
+SETLOCAL EnableExtensions
+
+ECHO Test return code success case
+call testclu success show 2> out.error
+REM We expect that the commands above succeed (return 0)
+IF %ERRORLEVEL% NEQ 0 (
+	ECHO "Expected error level 0 for success, got " %ERRORLEVEL%
```

Comment:
That is fine. To me it would be normal to swallow the stdout but still let stderr surface to the console, but it is your call.